frontend/app/components/ButtonOrLink.tsx

```diff
@@ -49,7 +49,9 @@ export const ButtonOrLink = forwardRef<
   const isLink = typeof to !== 'undefined' || typeof href !== 'undefined'
 
   if (isLink) {
-    return <AnchorOrLink href={href} to={to} ref={ref} {...props} />
+    return (
+      <AnchorOrLink href={href} to={to} ref={ref} {...props} target="_parent" />
+    )
```

Comment:
Would be better to do this in `TypeCard` instead, passing this `target="_parent"` to `Button` there. `ButtonOrLink` is too generic to add this behavior - it can bite us in future.